### PR TITLE
Update the German Translation

### PR DESCRIPTION
**Pull request by wheelseal**

### CODE REVIEW

File: translations/harbour-seaprint-de.ts [hunk 1]
```diff
@@ -613,1234 +613,1201 @@ auf diesem Drucker</translation>
...
     </message>
     <message>
         <source>10 x 11″</source>
-        <translation type="unfinished"></translation>
+        <translation>10 x 11″</translation>
     </message>
     <message>
-        <source>10 x 13″</source>
-        <translation type="unfinished"></translation>
+        <source>10 x 13 Zoll</source>
```
Comment (attah) [inline translations/harbour-seaprint-de.ts]: Please don't break the source translation.

**Comment by attah** [comment]
The missing lines and random characters are still not fixed.
Please _squash_ your changes and review them yourself first.
Consider that long translations will not fit the UI.
You have translated hagaki differently... not sure why it appears twice, could be for size and type perhaps.

**Review by attah — CHANGES_REQUESTED** [review]

File: translations/harbour-seaprint-de.ts (hunk 1 above)
Comment (wheelseal) [reply translations/harbour-seaprint-de.ts]: </message>
    <message>
        <source>10 x 13 Zoll</source>
        <translation>10 x 13 Zoll</translation>
    </message>
    <message>
nothing is broken

Comment (attah) [reply translations/harbour-seaprint-de.ts]: The translation is fine, but you changed the _source_.

**Pull request closed without merging**